src/styles/blocks/card.scss

```diff
@@ -0,0 +1,143 @@
+/* #region card */
+.card {
+  display: flex;
+  flex-direction: column;
+
+  margin: 0;
+  width: $card-width;
+
+  box-sizing: border-box;
+  border: $border-width solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+
+    margin-left: 19px;
+    margin-top: 32px;
+    margin-bottom: 24px;
+  }
+
+  &__product-item {
+    width: 166px;
+    height: 36px;
+    line-height: 18px;
+
+    margin-top: 16px;
+    margin-left: 16px;
+    margin-bottom: 4px;
```

Comment:
Don't use both margins (top and bottom), choose only one of them and use for all card elements